src/driver/src/DefaultDriver.cpp

```diff
@@ -77,27 +77,34 @@ void runActor(Actor&& actor,
     }
 }
 
-genny::driver::DefaultDriver::OutcomeCode doRunLogic(
-    const genny::driver::DefaultDriver::ProgramOptions& options) {
-    if (options.shouldListActors) {
-        globalCast().streamProducersTo(std::cout);
-        return genny::driver::DefaultDriver::OutcomeCode::kSuccess;
-    }
-
+DefaultDriver::OutcomeCode doRunLogic(const DefaultDriver::ProgramOptions& options) {
     genny::metrics::Registry metrics;
-
     auto actorSetup = metrics.operation("Genny", "Setup", 0u);
     auto setupCtx = actorSetup.start();
 
+    if (options.runMode == DefaultDriver::RunMode::kListActors) {
+        globalCast().streamProducersTo(std::cout);
+        setupCtx.success();
+        return DefaultDriver::OutcomeCode::kSuccess;
+    }
+
     auto yaml = loadConfig(options.workloadSource, options.workloadSourceType);
     auto orchestrator = Orchestrator{};
 
+    if (options.runMode == DefaultDriver::RunMode::kEvaluate) {
+        std::cout << "Printing evaluated workload YAML file:" << std::endl;
+        std::cout << YAML::Dump(yaml) << std::endl;
```

Comment:
I find it a little odd that I can run `./build/src/driver/genny --run-mode evaluate` and not get an error from not having specified a file.

Reply:
I added error handling for empty `workloadSource` when not in `list-actors` mode.